tests/suites/test_suite_ssl.function

```diff
@@ -157,9 +156,14 @@ typedef struct mbedtls_test_message_queue
     int capacity;
 } mbedtls_test_message_queue;
 
-
```

Comment:
This newline should be removed in "Add a message metadata queue in ssl tests", not "Add a message-based socket mock connection to the ssl tests"